kernel/build/config-amd64

```diff
@@ -786,10 +790,7 @@ CONFIG_ARCH_HAS_GCOV_PROFILE_ALL=y
 # end of GCOV-based kernel profiling
 
 CONFIG_HAVE_GCC_PLUGINS=y
-CONFIG_GCC_PLUGINS=y
```

Comment:
I think this is too much, we need to keep GCC_PLUGINS enabled, while (probably) disabling only the `CONFIG_GCC_PLUGIN_RANDSTRUCT=y`. Other gcc plugins provide meaningful security protection features which we shouldn't be dropping

Reply:
Sorry about the spam, accidentally pushed, you can cancel these 2 builds https://ci.dev.talos-systems.io/siderolabs/pkgs/1838
https://ci.dev.talos-systems.io/siderolabs/pkgs/1837